e2e/tests/gatekeepingOrderGenerated_test.js

```diff
@@ -5,7 +5,7 @@ const hearingDate = '10 January 2050, 3:15pm';
 
 let caseId;
 
-Feature('Gatekeeping judge SDO journey');
+Feature('Generated gatekeeping Order');
```

Comment:
vv minor but `order` instead of `Order`

Reply:
Also at the end of the existing set of tests in `gatekeeperAdministersCaseAfterGatekeeping_test.js` we assert that the case state has changes, do you that is worth having here?
```js
  caseViewPage.selectTab(caseViewPage.tabs.history);
  I.seeEndStateForEvent(config.administrationActions.draftStandardDirections, 'Case management');
```

Reply:
done